.github/workflows/codeql.yml

```diff
@@ -0,0 +1,44 @@
+name: "Code Scanning - Action"
+
+on:
+  push:
```

Comment:
We should probably remove the `push` event trigger which is covered by the regular CI workflow.